development-vm/README.md

```diff
@@ -1,276 +1,212 @@
+> You'll need to use a Mac to follow this guide.
 
-Either way, you will need virtualisation enabled in your BIOS, otherwise it
-won't work. This tends to be enabled by default on Macs, but is worth
-checking for other manufacturers.
+It's expected to take roughly a day to complete this guide, start to finish. There's a lot of things to download, and loads of installers need to do their thing.
 
-[1]: http://www.12factor.net/dev-prod-parity
-[2]: https://www.virtualbox.org/
-[3]: http://vagrantup.com/
-[4]: https://github.com/alphagov/wiki/wiki
+> You will need up to 50GB of free space on your hard-drive to run the whole of gov.uk.
 
-## 1. Prerequisites and assumptions
+### Developing outside the VM
 
-  * You have [OSX GCC tools installed](https://github.com/kennethreitz/osx-gcc-installer) (or via XCode)
-  * You have [VirtualBox](https://www.virtualbox.org/) installed
-  * You have [Vagrant](https://www.vagrantup.com/downloads.html) installed
-  * you have vagrant-dns plugin installed ` vagrant plugin install vagrant-dns `
-  * You have an account on [GitHub.com](https://github.com)
-  * You have some other repositories from GOV.UK checked out to work on -
-    these should be located alongside the `govuk-puppet` repository we're in
-    now (e.g `~/govuk/govuk-puppet:~/govuk/frontend`)
+You don't have to develop on the VM, but we strongly recommend it. If you have problems with the development VM you can always ask for help in the #govuk-developers Slack channel.
 
-### A note on Boxen
+### Where you should run commands
 
-If you have a GDS issued laptop and you want to automate most of this,
-consider building your laptop using
-[GDS Boxen](https://github.com/alphagov/gds-boxen). Specifically
-you'll want these things:
+Run mac$ commands in the shell on your Mac:
 
-    include vagrant
-    include virtualbox
-    vagrant::plugin { 'vagrant-dns': }
-    include projects::puppet
+    mac$ echo "Think Different"
 
-If your profile is based on
+Run dev$ commands in the shell on the development VM:
 
-    include gds-development
+    dev$ echo "Linux for human beings"
 
-then you will already have these, and will automatically get any
-improvements made in that module. This is the recommended approach.
+## 1. Install some dependencies
 
-## 2. Booting your VM
+First, install:
 
-Assuming your Puppet code is checked out into:
+* [OSX GCC tools](https://github.com/kennethreitz/osx-gcc-installer)
+* [VirtualBox](https://www.virtualbox.org/)
+* [Vagrant](https://www.vagrantup.com/downloads.html)
+* vagrant-dns plugin: `vagrant plugin install vagrant-dns`
 
-    ~/govuk/govuk-puppet
+## 2. Create your GitHub accounts
 
-the following commands will bootstrap your VM:
+1. Set up a [GitHub](https://www.github.com) account.
+1. Get yourself added to the [alphagov organisation](https://github.com/alphagov). Your tech lead will be able to facilitate this.
+1. Set up a GitHub Enterprise (private) account and be added to the `gds` organisation.
```

Comment:
Set up a GitHub Enterprise (private) account and ask to be added to the `gds` organisation.